templates/commandline.tmpl

```diff
@@ -57,14 +57,14 @@ $ sudo mv ./kubectl /usr/local/bin/kubectl
                <code class="language-bash">
 echo "{{ .ClusterCA }}" \ > ca-{{ .ClusterName }}.pem
 kubectl config set-cluster {{ .ClusterName }} --server={{ .APIServerURL }} --certificate-authority=ca-{{ .ClusterName }}.pem --embed-certs
-kubectl config set-credentials {{ .Username }}@{{ .ClusterName }}  \
+kubectl config set-credentials {{ .Email }}  \
```

Comment:
@paulczar in the issue we discussed making this `user@cluster`?

Reply:
yes, which it is ...  just its doing it in the logic that was fetching emailClaim rather than in the template.  see [here](https://github.com/heptiolabs/gangway/pull/87/files#diff-b0b3c2b4705fd8725517237908398533R236).    Instead of completely ignoring emailClaim it will use it if provided in a config (with a deprecation warning ) and if its not set it sets the existing `email` field to be username@cluster.

Thus both the new and the old work, and the deprecation can be remove later at a suitable time with a major version cut.

Reply:
Ahh I see. My only nit is that it's confusing if you just look at the template file. Maybe a different variable name would make sense just to make it clear?